src/chatbot.py

```diff
@@ -101,8 +101,11 @@ def chat(encoder, decoder, voc, speaker_name):
     decoder.eval()
 
     # Initialize search module
-    speaker_id = voc.people2index[speaker_name]
-    searcher = GreedySearchDecoder(encoder, decoder, speaker_id)
+    if config.BEAM_SEARCH_ON:
+        searcher = BeamSearchDecoder(encoder, decoder)
```

Comment:
If you use the new DecoderRNN as your input decoder, you may need to add the speaker_id tensor, since it's a required param for the persona decoder.

Reply:
Yes, I will add it later, thanks.